### ISSUE TITLE
[BUG?] Not translating specific template fields

### ISSUE THREAD

**Issue by Sendoushi** [AnimalDesign/bolt-translate#117]
Using a specific templatefield, I set the "title" to "X" in a language A and to "Y" in language B. Language A "title" has become "Y" when it should've stayed at "X".

Details
-------

 - Translate Version: 4.0.5
 - Install type: Extension Repository
 - Bolt Version: 3.2.11
 - PHP version: 7.0
 - Used webserver: Nginx

Reproduction
------------

 1. **Blocks content type definition** (main type): 

```
blocks:
    name: Blocos
    singular_name: Bloco
    slug: blocks
    singular_slug: block
    fields:
        title:
            type: text
            class: large
            group: "Bloco"
            is_translateable: true
        slug:
            type: slug
            uses: title
        template:
            type: templateselect
            filter: 'view/blocks/*.twig'
            required: true
        # Translations
        locale:
            type: locale
        ptdata:
            type: hidden
        ptslug:
            type: locale_data
            index: true
        frdata:
            type: hidden
        frslug:
            type: locale_data
            index: true
    show_on_dashboard: true
    viewless: true
    default_status: publish
    searchable: false
    icon_many: "fa:cubes"
    icon_one: "fa:cube"
```

 2. **Template fields definition** (where the problem resides): 

```
templatefields:
    view/blocks/hero.twig:
        title:
            type: html
            class: large
            is_translateable: true
```

 3. Set a new block with template on `hero.twig`
 4. On the specific template field of the hero, `title`, set `X`
 5. Change to language `B` and the `title`, to `Y`
 6. Change back to language `A` and notice that now `title` is `Y` instead of `X`

**Comment by benwallis** [comment]
Check if it works on the front-end. 

It might be the bug where correct translation is not shown in the admin:
https://github.com/AnimalDesign/bolt-translate/issues/110

**Comment by lenvanessen** [comment]
You need to tag the `templateselect` with is_translateable instead of the individual fields.

Like:
```
 template:
            type: templateselect
            filter: 'view/blocks/*.twig'
            required: true
```

Updated readme is coming in that clarify's this.

**Comment by SahAssar** [comment]
Fixed by a readme update in 10a40d0
